File: Team4/Nikita/XorOperation.cpp
class Solution {
public:
    int xorOperation(int n, int start) {
     
        int nums[n] ;
        nums[0] = start;
        int k = nums[0];
        for(int i = 1 ; i < n ; i++)
        {
            nums[i] = start + 2*i;
            k ^= nums[i];
        }
        
        return k;
    }
};

File: Team2/kusumlata/XORoperation.java
class Solution {
    public int xorOperation(int n, int start) {
        int[] arr = new int[n];
        for(int i = 0;i<n;i++){
            arr[i] = start;
            start = start + 2;
        }
        int res = arr[0];
        for(int i = 1;i<arr.length;i++){
            res = res ^ arr[i];
        }
        return res;
    
        
    }
}
